runtime/browser-vr-demo/recipes.js

```diff
@@ -0,0 +1,102 @@
+module.exports = [{
```

Comment:
I expect this file to be completely different when I get back to working on it. I think the idea at this stage is to demo different use cases for VR/AR.